src/transformers/models/speech_encoder_decoder/modeling_flax_speech_encoder_decoder.py

```diff
@@ -108,8 +108,7 @@
             If `past_key_values` is used, optionally only the last `decoder_input_ids` have to be input (see
             `past_key_values`).
 
-            For training, `decoder_input_ids` are automatically created by the model by shifting the `labels` to the
-            right, replacing -100 by the `pad_token_id` and prepending them with the `decoder_start_token_id`.
```

Comment:
Think we should keep this comment but just mention that this needs to be done outside of the model and does not happen automatically. Same for the `FlaxEncoderDecoder` model.